src/app/static/src/js/components/runReport/changeLog.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div v-if="showChangeMessage">
+        <div id="changelog-message" class="form-group row">
+            <label for="changelogMessage"
+                   :class="`col-sm-${changelogStyle.label.size} ${changelogStyle.label.justify}`"
+                   class="col-form-label">Changelog Message</label>
+            <div :class="`col-sm-${changelogStyle.control.size}`">
+                        <textarea class="form-control" id="changelogMessage"
+                                  v-model="changeLogMessageValue"
+                                  @input="handleChangeLogMessage"
+                                  rows="2">
+                        </textarea>
+            </div>
+        </div>
+        <div id="changelog-type" class="form-group row">
+            <label for="changelogType"
+                   :class="`col-sm-${changelogStyle.label.size} ${changelogStyle.label.justify}`"
+                   class="col-form-label">Changelog Type</label>
+            <div :class="`col-sm-${changelogStyle.control.size}`">
+                <select class="form-control"
+                        id="changelogType"
+                        v-model="changeLogTypeValue"
+                        @change="handleChangeLogType">
+                    <option v-for="option in reportMetadata.changelog_types" :value="option">
+                        {{ option }}
+                    </option>
+                </select>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import {ChangelogStyle, ReportMetadata} from "../../utils/types";
+
+    interface Props {
+        showChangeMessage: boolean
+        reportMetadata: ReportMetadata[] | null
```

Comment:
This looks strange: a) why a `[]`, and b) why `null`? But may be a moot point given my comment below.